esp-hal/src/spi/master.rs

```diff
@@ -983,7 +1015,13 @@ where
     }
 
     /// Change the bus configuration.
-    // FIXME: when https://github.com/esp-rs/esp-hal/issues/2839 is resolved, add an appropriate `# Error` entry.
+    ///
+    /// # Errors.
+    /// If frequency passed in config exceeds
```

Comment:
```suggestion
    /// # Errors
    ///
    /// If frequency passed in config exceeds
```

Add empty line and remove the final dot